### PR TITLE
include UI for statistics experience

### PR DESCRIPTION
**Pull request by CarLosVegga**
Hi everyone! 

I was working a bit on how I wanted to display my statistics data for an experience maker.

I used dummy data to see how easy/complicated it would be to use victory. It turned out well, and here are the results.

## EXAMPLE
image

On feedback Area I will include the name of the feedback area and whether it is active feedback (which means open to reviews).

### Experience.jsx
I deleted a value I was retrieving from a context, and called the Performance component to render this page.

### Comment.jsx
I included the necessary libraries, and using dummy data I started playing with the UI to have a general idea on how to display the comments. 

### Graph.jsx
Again I used dummy data and also played with some CSS elements, used transitions and played with the colors to differentiate between how good certain bar is according to what the user selected.

### Performance.jsx
The plan is that, from line 26 to 33 I will repeat those elements. For that, I was thinking on using a forEach/map function to go over the feedback of every feedback information. I also created a function to control the colors of the UI according to the score that we are dealing with: 1 (bad) - red, 5 (great) blue

### NEXT STEPS
- I will implement the endpoints to get the real information and probably refactor a bit of code according to how the back-end needs to adapt my front-end
- I also wanted some feedback on how to polish my UI. Anything would be deeply appreciated!

### CODE REVIEW

File: capstone/src/components/User/Experience/Performance/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Box, Badge, Text } from '@chakra-ui/react';
+
+const data = [
```
Comment (vqdo) [inline capstone/src/components/User/Experience/Performance/Comment.jsx:3]: Great idea to use placeholder data.

File: capstone/src/components/User/Experience/Performance/Comment.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  { score: 1, comment: ['really bad food'] },
+  { score: 2, comment: ['not that bad', 'ok if you are really hungry', 'need more work'] },
+  { score: 3, comment: ['can improve but ok'] },
+  { score: 4, comment: ['it was ok'] },
+  { score: 5, comment: ['my favorite place', 'it changed my life'] },
+];
+
+export default function Comment({determineColor}) {
```
Comment (vqdo) [inline capstone/src/components/User/Experience/Performance/Comment.jsx:11]: It looks like we're rendering multiple comments. Maybe we should name this `CommentArea` or `CommentSection` to be clearer?

File: capstone/src/components/User/Experience/Performance/Performance.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    case 4:
+      return '#68D391';
+    case 5:
+      return '#38B2AC';
+    default:
+      return '#0000000';
+  }
+}
+
+export default function Performance() {
```
Comment (vqdo) [inline capstone/src/components/User/Experience/Performance/Performance.jsx:22]: I guess these are copied from the library examples, but it'd be good to use names specific to this app.

**Review by vqdo — APPROVED** [review]

File: capstone/src/components/User/Experience/Performance/Comment.jsx (hunk 1 above)
Comment (CarLosVegga) [reply capstone/src/components/User/Experience/Performance/Comment.jsx]: Thanks!

File: capstone/src/components/User/Experience/Performance/Comment.jsx (hunk 2 above)
Comment (CarLosVegga) [reply capstone/src/components/User/Experience/Performance/Comment.jsx]: Definitely! I will add it in the upcoming commit

File: capstone/src/components/User/Experience/Performance/Graph.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { VictoryChart, VictoryBar, VictoryLabel } from 'victory';
```
[the remark this answers is not in the document]
Comment (CarLosVegga) [reply capstone/src/components/User/Experience/Performance/Graph.jsx:1]: Alright!

File: capstone/src/components/User/Experience/Performance/Performance.jsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import { Box, HStack, Heading, Text } from '@chakra-ui/react';
+import Graph from './Graph';
+import Comment from './Comment';
+
+function determineColor(value) {
```
[the remark this answers is not in the document]
Comment (CarLosVegga) [reply capstone/src/components/User/Experience/Performance/Performance.jsx:15]: That makes more sense!

**Pull request merged**